install/deployment_yamls/istio.yaml

```diff
@@ -25,10 +27,10 @@ spec:
   http:
   - match:
     - uri:
-        prefix: /
+        prefix: /meshery
```

Comment:
@samirparhi-dev Any reason why prefix is set to `/meshery`

Reply:
This is because, if my ingress gateway has the FQDN ( lestsay : myapplication.com), which serves all the service from my k8s ( which is a best practice in production grade cluster). I am supposed to access meshery with. myapplication.com/meshery. Insted exposing individual loadbalancer ip to outside world .

Reply:
Gents, was there any further discussion had here?

Reply:
@leecalcote : I was waiting for the review and response.

Reply:
@samirparhi-dev I think this is something specific to your usecase, there could be more others who will be using multiple endpoints with separate FQDNs. Because meshery is going to manage multiple service meshes, I think a separate FQDN is an ideal design. // @leecalcote thoughts?

Reply:
@kumarabd waiting for your response here.

Reply:
Err... no, we're not waiting on a response from @kumarabd. 😆 I missed your last statement and question, @kumarabd.

I agree: `prefix` is an environment-specific concern that we would do well not to predefine. @samirparhi-dev, assuming you're ok with that perspective, will you update this line? And we'll proceed with the other changes.

Reply:
Yes , I will change that